packages/at_onboarding_flutter/lib/at_onboarding.dart

```diff
@@ -69,49 +69,57 @@ class AtOnboarding {
         return AtOnboardingResult.cancelled();
```

Comment:
For this file, there are a lot of repetitive cancels, and a lot of nested if statements. Can we try to use the guard clause pattern to clean this up and make it easier to follow.

Reply:
Refactored code to use guard clause pattern for improved readability and reduced nesting